tensorflow/c/tf_tensor.h

```diff
@@ -45,6 +45,16 @@ limitations under the License.
 extern "C" {
 #endif
 
+// Allocator Attributes used for tensor allocation 
+struct TF_AllocatorAttributes { 
+  size_t struct_size; 
+  unsigned char on_host; 
+} TF_AllocatorAttributes; 
+
+
+#define TF_ALLOCATOR_ATTRIBUTES_STRUCT_SIZE \
+    TF_OFFSET_OF_END(TF_AllocatorAttributes, on_host) 
```

Comment:
Where is `TF_OFFSET_OF_END` defined, and does tensorflow/c/tf_tensor.h pull in that dependency?

Reply:
Thanks for pointing it out. Just added it to the file!